source/blog/2019-04-15-ovn-security-groups.html.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Security group support in OVN external networks
+author: mdbarroso
+tags: Open vSwitch, OVN, ansible, Acess Control
+date: 2019-04-15 15:00:00 CET
+comments: true
+published: true
+---
+
+In this post I will introduce and showcase how security groups can be used to
+enable certain scenarios.
+
+[Security groups](https://developer.openstack.org/api-ref/network/v2/#security-groups-security-groups) allow fine-grained access control to - and from - the oVirt VMs attached to external OVN networks.
+
+The Networking API v2 defines security groups as a white list of rules - the
+user specifies in it which traffic is allowed. That means, that when the rule
+list is empty, neither incoming nor outgoing traffic is allowed (from the VMs
+perspective).
+
+READMORE
+
+## Provided tools
```

Comment:
I am not sure if all the details are useful here, or if a link to your text on github would be sufficient, but the text does not hurt, as long as both videos are embedded above.

Reply:
Linked against hte original README.